src/rust/aggregator/src/main.rs

```diff
@@ -22,9 +22,9 @@ async fn main() -> Result<()> {
 
     let mut data: Vec<Arc<Mutex<dyn Data + Send + Sync>>> = vec![];
 
-    data.push(Arc::new(Mutex::new(MarketsRegisteredPerDay::new(
-        pool.clone(),
-    ))));
+    // data.push(Arc::new(Mutex::new(MarketsRegisteredPerDay::new(
```

Comment:
Why is this commented out, and if so, why isn't it deleted instead?
Commented out code is a smell

Reply:
Big smell indeed. I did this for testing, forgot to undo. Thanks for spotting.